app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -313,7 +314,11 @@ class ProfileEditActivityTest {
         .inRoot(isDialog())
         .perform(click())
       testCoroutineDispatchers.runCurrent()
-      intended(hasComponent(ProfileListActivity::class.java.name))
+      if (context.resources.getBoolean(R.bool.isTablet)) {
+        intended(hasComponent(AdministratorControlsActivity::class.java.name))
+      } else {
+        intended(hasComponent(ProfileListActivity::class.java.name))
+      }
     }
   }
```

Comment:
Also update the other test case `testProfileEdit_configChange_startWithUserProfile_deleteProfile_checkReturnsToProfileList`

Reply:
Done

Reply:
We can skip the startWithProfile to avoid ktlint disable.

Reply:
![image](https://user-images.githubusercontent.com/54740946/100686368-30202c00-33a4-11eb-906d-3d1bd3908f9a.png)

Reply:
it exceeds even after removing startWithUserProfile

Reply:
`testProfileEdit_configChange_deleteProfile_checkReturnsProfileListOnTabletAdminControlOnPhone` could you check this one!!

Reply:
@anandwana001 it's Close but still exceeds
![image](https://user-images.githubusercontent.com/54740946/100687526-7bd3d500-33a6-11eb-90ce-a75fe13286aa.png)

Reply:
ah, could you suggest anything better, we are trying to avoid such ktlint disabling.

Reply:
testProfileEdit_landscape_deleteProfile_checkReturnsProfileListOnTabletAdminControlOnPhone this works

Reply:
sounds good.

Reply:
Done